strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/dependency/snippet/PypiWheelDependencyFormatter.java

```diff
@@ -0,0 +1,65 @@
+package org.carlspring.strongbox.dependency.snippet;
+
+import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
+import org.carlspring.strongbox.artifact.coordinates.PypiWheelArtifactCoordinates;
+import org.carlspring.strongbox.providers.layout.AbstractLayoutProvider;
+import org.carlspring.strongbox.providers.layout.PypiLayoutProvider;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+* This class is an implementation of DependencySynonymFormatter for Pypi artifacts
+*
+* @author whalenda
+**/
+@Component
+public class PypiWheelDependencyFormatter
+  implements DependencySynonymFormatter
+{
```

Comment:
Cosmetic: Could you please follow the project's [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html) and reformat all your changes to conform with it? Thanks! :)